packages/twenty-front/src/modules/object-record/record-table/components/RecordTableNoRecordGroupRows.tsx

```diff
@@ -15,7 +15,8 @@ export const RecordTableNoRecordGroupRows = () => {
           <RecordTableRow
             key={recordId}
             recordId={recordId}
-            rowIndex={rowIndex}
+            rowIndexForFocus={rowIndex}
```

Comment:
I'm introducing two rowIndexes:
- one is used to handle focus, the other to give an index for drag

Right now the index used in focus / cell opening is handled at table level so it's a global index, the index use drag is at group level. I think long term we want to have the index for focus / cell opening at group level too and handle the jump between groups properly

I prefer introducing these two naming ForFocus and ForDrag because it's clear when to use which one in underlying components. Alternatively I could have introduced rowIndexInGroup and rowIndexInTable but this will make underlying components know about recordGroups and make if conditions which I think we should avoid to reduce complexity